tests/external/test_report_downloader.py

```diff
@@ -63,3 +63,27 @@ def test_invalid_provider_type(self):
                                  report_source='hereiam',
                                  report_name='bestreport',
                                  provider_type='unknown')
+
+    @patch('masu.external.downloader.aws.aws_report_downloader.AWSReportDownloader.__init__', return_value=None)
+    def test_get_current_report(self, fake_downloader):
+        """Test get_current_report function."""
+        downloader = ReportDownloader(customer_name='customer name',
+                                      access_credential='mycred',
+                                      report_source='hereiam',
+                                      report_name='bestreport',
+                                      provider_type=AMAZON_WEB_SERVICES)
+        with patch.object(AWSReportDownloader, 'download_current_report', return_value=self.file_list):
+            files = downloader.get_current_report()
+            self.assertEqual(len(files), len(self.file_list))
+
+    @patch('masu.external.downloader.aws.aws_report_downloader.AWSReportDownloader.__init__', return_value=None)
+    def test_get_current_report_error(self, fake_downloader):
+        """Test get_current_report function with error."""
+        downloader = ReportDownloader(customer_name='customer name',
+                                      access_credential='mycred',
+                                      report_source='hereiam',
+                                      report_name='bestreport',
+                                      provider_type=AMAZON_WEB_SERVICES)
+        with patch.object(AWSReportDownloader, 'download_current_report', side_effect=Exception('some error')):
+            with self.assertRaises(ReportDownloaderError):
+                _ = downloader.get_current_report()
```

Comment:
unused-variable

Reply:
what variable is unused?

Reply:
The `_ =`.  That's an anti-pattern. Please stop using it.

Reply:
umm ok..  It's nice to know that the function returns something but you don't care what it is.  What's so wrong with that?